scanomatic/models/validators/tools.py

```diff
@@ -0,0 +1,107 @@
+from enum import Enum
```

Comment:
most things in here were just moved here, but I wrote tests for them anyhow since it seemed a bit more important than the specific validator functions.